shard.yml

```diff
@@ -1,9 +1,14 @@
 name: wordsmith
-version: 0.3.0
+version: 0.3.2
 
 authors:
   - Paul Smith <paulcsmith0218@gmail.com>
   - Flinn Mueller <theflinnster@gmail.com>
+  - Billy.Zheng <vil963@gmail.com>
```

Comment:
Although I do appreciate the contribution, I leave this section for the original creators only :sweat_smile:  otherwise my name would also be in here :smile:

Reply:
Done in 7b20234